OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -3412,5 +3413,131 @@ public void TestAfterDisposeAPIsShouldNotCrash()
         }
 
         #endregion
+
+        #region Test GetOptimizelyConfig
+        [Test]
+        public void TestGetOptimizelyConfig()
+        {
+            ProjectConfig actualConfig = OptimizelyMock.Object.ProjectConfigManager?.GetConfig();
+            OptimizelyConfig optimizelyConfig = OptimizelyMock.Object.GetOptimizelyConfig();
+            Assert.AreEqual(optimizelyConfig.Revision, actualConfig.Revision);
+            foreach (var actualEx in actualConfig.Experiments)
```

Comment:
Please revise it. Define the hardcoded`Object` and compare actual object using `TestData.Compare`